src/neuigkeiten/2024-01-06-zeiterfassung-2.0.0/index.md

```diff
@@ -0,0 +1,105 @@
+---
+draft: true
+layout: layout-neuigkeiten
+date: 2024-01-06
+title: Arbeitszeiten, Feiertagsregelung und mehr Berechtigungen in der Zeiterfassung
+author: Benjamin Seber
+release-version: 2.0.0
+teaserImage: /static/zeiterfassung-multi-device.png
+description: Kroatische Feiertage für eine einfache Urlaubsplanung
+---
+
+Zum Jahresstart gibt es spannende Aktualisierungen bei der Zeiterfassung:
+
+- flexible Arbeitszeiten für Vertragsänderungen
+- Bundesländer und Feiertagsregelungen individuell anpassen
+- Berechtigungen direkt in der Anwendung vergeben
+
+<!-- more -->
+
+## Flexible Arbeitszeiten abbilden
+
+Ab sofort könnt ihr mühelos mehrere Arbeitszeiten erstellen und verwalten. Sei es aufgrund von Teilzeitvereinbarungen,
+Vollzeitbeschäftigungen oder anderen Vertragsänderungen. Die Anwendung gibt euch auf der Personenübersichtseite eine
+kompakte Ansicht damit ihr stets den Überblick behaltet.
+
+<div class="my-8 flex flex-col gap-4 md:flex-row">
+    <figure>
+        <picture>
+            <source srcset="arbeitszeiten.avif" type="image/avif" />
+            <img
+              src="arbeitszeiten.png"
+              alt="Weitere Arbeitszeiten erstellen und verwalten für Vertragsänderungen."
+              width="1440"
+              height="1156"
+              decoding="async"
+              loading="lazy"
+              class="rounded-lg"
+            />
+        </picture>
+        <figcaption class="text-sm text-center">Weitere Arbeitszeiten erstellen und verwalten für Vertragsänderungen.</figcaption>
+    </figure>
+    <figure>
+        <picture>
+            <source srcset="arbeitszeiten_edit.avif" type="image/avif" />
+            <img
+              src="arbeitszeiten_edit.png"
+              alt="Weitere Arbeitszeiten erstellen und verwalten für Vertragsänderungen."
+              width="1440"
+              height="1157"
+              decoding="async"
+              loading="lazy"
+              class="rounded-lg"
+            />
+        </picture>
+        <figcaption class="text-sm text-center">Bundesland und Feiertagsregelung individuell pflegen.</figcaption>
+    </figure>
+</div>
+
+## Individuelle Feiertagsregelung pro Bundesland
+
+Bei verteilten Teams ist es nicht selten, dass unterschiedliche Anforderungen an Bundesländer und Feiertage existieren.
+Für Personen, die in Baden-Württemberg angestellt sind, gelten andere Feiertage als z. B. in Hessen. Daher könnt ihr ab
+sofort das Bundesland und die Feiertagsregelung für jede Person individuell anpassen.
+
+Die Zeiterfassung unterstützt dabei nicht nur deutsche Bundesländer, sondern auch viele andere Länder aus Europa.
```

Comment:
Wollen wir drauf eingehen, dass die Feiertagsregelung global zu steuern in der nächsten Version kommt?

Reply:
hm, hätte ich nicht. aber kann dir keinen grund dafür sagen ^^" vielleicht meldet sich ja jemand und wir freuen uns, dass es wirklich gewünscht wird 🙃